sklearn/decomposition/pca.py

```diff
@@ -488,7 +488,8 @@ class RandomizedPCA(BaseEstimator, TransformerMixin):
         use fit_transform(X) instead.
 
     iterated_power : int, optional
-        Number of iterations for the power method. 3 by default.
+        Number of iterations for the power method. 2 by default.
+        .. versionchanged:: 0.18
```

Comment:
That's indeed the case. Fixed now.

![screen shot 2015-10-21 at 6 33 19 pm](https://cloud.githubusercontent.com/assets/2871319/10643278/49aa0a4e-7822-11e5-8c74-f84280cd4bc1.png)